packages/sync/README.md

```diff
@@ -1,19 +1,17 @@
-## AeroGear Services Sync SDK
+# AeroGear Apollo GraphQL Client
```

Comment:
Description in the `package.json` says: _AeroGear Apollo GraphQL Voyager client_